app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/YouCannotUseServiceController.scala

```diff
@@ -46,8 +46,9 @@ class YouCannotUseServiceController @Inject() (
       } recover withAuthRecovery(request)
   }
 
-  def unauthorisedPage(): Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Unauthorized(unauthorisedView()))
+  def unauthorisedPage(service: Service, journey: Journey.Value): Action[AnyContent] = Action.async {
+    implicit request =>
+      Future.successful(Unauthorized(unauthorisedView()))
```

Comment:
You can remove `.async` and `Future.successful`.

Async is only required when there is a future inside. If you don't have any futures in body, it's enough to use just Action and provide a result inside.

Both approaches for Actions are asynchronous.

Edit: But as I notice it's old implementation